src/components/SelectButtonGroup/SelectButton/SelectButton.tsx

```diff
@@ -20,28 +18,19 @@ export interface Props<V> extends InternalProps<V> {
     children: NotEmptyReactNode;
     /** the value associated with this button */
     value: V;
-    /** if this button should be in a selected when first rendered */
-    isInitiallySelected?: boolean;
+    /** whether or not this button is selected */
+    isSelected?: boolean;
     /** the color context to be applied in the selected state */
-    selectedContext?: Context;
+    context?: Context;
```

Comment:
The reason is that it is more consistent with use of `context` in any other component. This weighs more in my opinion and since there would be other breaking changes, might as well rename it now.

Reply:
I think we can get away without breaking changes

Reply:
See https://github.com/textkernel/oneui/pull/624#discussion_r743715669